packages/storefront-lwc/src/modules/commerce/contextHeading/contextHeading.scss

```diff
@@ -0,0 +1,66 @@
+@import "../../../scss/ux_variables";
+
+.sub-header{
+    height: 4.688em;
+  
+    @include mobile {
+      height: 3.125em;
+      margin-bottom: -1em;
+    }
+
+    @include desktop {
+      margin-top: 1em;
+      height: 5em;
+    }
+  }
+  
+  .sub-header:before {
+    content: '';
+    position: absolute;
+    top: 50%;
+    left: 0;
+    border-top: 1px solid $light-gray;
+    background: $light-gray;
+    width: 100%;
+    transform: translateY(-50%);
+  }
+  
+  .sub-header-content {
+    background-color: white;
+    width: auto;
+    max-width: 90%;
+    overflow: hidden;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
Is the position absolute, transform, left, and top to center the content? There are easier ways to do this. For example, `display: flex; justify-content: center; align-items: center;`

Reply:
Flexbox (and css grid) really change the way CSS is written. I highly recommend taking some time to learn it. Here is a free (and fun) course that a friend of mine built: https://geddski.teachable.com/p/flexbox-zombies

Reply:
Thank you @blittle for sharing link! Taking the tutorial.  It's fun, as you said, and educational!

Reply:
Spoke to @blittle.  He is ok with this styling.